master/buildbot/manhole.py

```diff
@@ -13,6 +13,7 @@
 #
 # Copyright Buildbot Team Members
 from __future__ import print_function
+from builtins import str
```

Comment:
Or maybe I'm confusing things a little. I'll get back to you on that one after i have had some sleep :p